conf/checksums

```diff
@@ -4,22 +4,15 @@ fluo:1.2.0:037f89cd2bfdaf76a1368256c52de46d6b9a85c9c1bfc776ec4447d02c813fb2
 fluo_yarn:1.0.0:c6220d35cf23127272f3b5638c44586504dc17a46f5beecdfee5027b5ff874b0
 hadoop:3.3.1:2fd0bf74852c797dc864f373ec82ffaa1e98706b309b30d1effa91ac399b477e1accc1ee74d4ccbb1db7da1c5c541b72e4a834f131a99f2814b030fbd043df66
 hadoop:3.2.2:054753301927d31a69b80be3e754fd330312f0b1047bcfa4ab978cdce18319ed912983e6022744d8f0c8765b98c87256eb1c3017979db1341d583d2cee22d029
-hadoop:3.1.4:f7eff67e948e21e23d7069dc169c98fcce757f59473e1e9a0e27d69a23df88840a9776089eeae5a8e1e496753f811868fe65638b04e3388c197931cd44d86f63
```

Comment:
This should be kept. It's the latest in the 3.1 series and we still do release testing against this version for Accumulo

Reply:
This (and the other ones I removed) are no longer available for download from the regular mirrors. If you try deploying a cluster on a completely new environment where the tarball was not already downloaded earlier, the download task (and hence setup) fails with a 404:

```
failed: [test-proxy] (item={u'sum': u'sha512:f7eff67e948e21e23d7069dc169c98fcce757f59473e1e9a0e27d69a23df88840a9776089eeae5a8e1e496753f811868fe65638b04e3388c197931cd44d86f63', u'urlp': u'https://dlcdn.apache.org//hadoop/common/hadoop-3.1.4', u'fn': u'hadoop-3.1.4.tar.gz'}) => {"ansible_loop_var": "item", "attempts": 3, "changed": false, "dest": "/home/centos/tarballs/hadoop-3.1.4.tar.gz", "elapsed": 0, "item": {"fn": "hadoop-3.1.4.tar.gz", "sum": "sha512:f7eff67e948e21e23d7069dc169c98fcce757f59473e1e9a0e27d69a23df88840a9776089eeae5a8e1e496753f811868fe65638b04e3388c197931cd44d86f63", "urlp": "https://dlcdn.apache.org//hadoop/common/hadoop-3.1.4"}, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "https://dlcdn.apache.org//hadoop/common/hadoop-3.1.4/hadoop-3.1.4.tar.gz"}
```

However, I'll look through if we can modify #427 to use these checksums along with a local tarball. That way, retaining these checksums will be useful.

Reply:
@ctubbsii - addressed your comments for this PR.